ci/run-validator.sh

```diff
@@ -67,7 +67,7 @@ provision_da_nodes() {
       echo "Creating a new keys for the $node_name"
       celestia-appd keys add "$node_name" --keyring-backend "test"
       # export it
-      echo "password" | celestia-appd keys export "$node_name" 2> "$key_file.lock"
+      echo "password" | celestia-appd keys export "$node_name" > "$key_file.lock"
```

Comment:
Between 3.0.2 and 3.4.2 celestia-appd switched from printing the key to stderr to stdout.